src/control/system/raft/database.go

```diff
@@ -559,9 +561,11 @@ func (db *Database) GroupMap() (*GroupMap, error) {
 		}
 
 		gm.RankEntries[srv.Rank] = RankEntry{
-			PrimaryURI:    srv.PrimaryFabricURI,
-			SecondaryURIs: srv.SecondaryFabricURIs,
-			Incarnation:   srv.Incarnation,
+			PrimaryURI:       srv.PrimaryFabricURI,
+			NumPrimaryCtxs:   srv.FabricContexts,
```

Comment:
Nit, but to be consistent should be prefixed with Primary ?

Reply:
Good catch, you're right, I will update the name.